Tests/Opc.Ua.Client.Tests/ClientFixture.cs

```diff
@@ -283,7 +284,28 @@ public ISession CreateSession(ITransportChannel channel, ConfiguredEndpoint endp
         {
             if (endpoints == null)
             {
-                endpoints = await GetEndpoints(url).ConfigureAwait(false);
+                try
+                {
+                    endpoints = await GetEndpoints(url).ConfigureAwait(false);
+                }
+                catch(Exception)
+                {
+                    // Don't ignore if platform is Windows
```

Comment:
any idea yet for the root cause?

Reply:
It looks as a TLS versioning mismatch on Linux (Ubuntu) even if the HttpsTransportListener and HttpsTransportChannel use the same TLS version

Reply:
did you check which endpoints are offered by https? At least one for RSA and every ECC profile should be included.

Reply:
The RSA is included, (net48) Tests-Opc.Ua.Client.Tests-windows passes when run locally but times out after exactly 30 s on the pipeline, under my Ubuntu dev machine GetEndpoint calls fail with TLS version error which I saw also on the pipeline run logs earlier